packages/sdk/src/react/provider.tsx

```
@@ -0,0 +1,73 @@
+ * @param props.burnerManager - Instance of BurnerManager for handling burner accounts
+ * @throws {Error} If DojoProvider is used more than once in the component tree
+ */
+export function DojoSdkProvider<Schema extends SchemaType>({
+    dojoConfig,
+    sdk,
+    clientFn,
+    children,
+}: {
+    dojoConfig: DojoConfig;
+    sdk: SDK<Schema>;
+    clientFn: Function;
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)